src/MessagePack.SourceGenerator/CodeAnalysis/FullModel.cs

```diff
@@ -58,6 +58,14 @@ public static FullModel Combine(ImmutableArray<FullModel> models)
             }
         }
 
+        var unboundGenericTypes = new HashSet<string>(
```

Comment:
I'd love to see a code comment explaining why these added operations exist.

Reply:
I reverted the change of `FullModel`. The modified code doesn't need it anymore.